docs/overrides/icons/kubernetes.svg

```diff
@@ -1 +1,6 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="-0.17 0.08 230.10 223.35">
```

Comment:
Not sure why but the text in the `<style>` tags was rendering. Have just moved the fill to the paths.